.pre-commit-config.yaml

```diff
@@ -10,9 +10,11 @@ repos:
         # exclude helm chart templates
         exclude: ^projects/control-service/projects/helm_charts
       - id: check-json
+        exclude: (^projects/frontend/data-pipelines/gui/tsconfig.json|^projects/frontend/data-pipelines/gui/package-lock.json|^projects/frontend/data-pipelines/gui/projects/data-pipelines/.eslintrc.json|^projects/frontend/data-pipelines/gui/projects/data-pipelines/tsconfig.lib.prod.json|^projects/frontend/data-pipelines/gui/projects/data-pipelines/tsconfig.spec.json|^projects/frontend/data-pipelines/gui/projects/data-pipelines/tsconfig.lib.json|^projects/frontend/data-pipelines/gui/projects/ui/.eslintrc.json|^projects/frontend/data-pipelines/gui/projects/ui/tsconfig.app.json|^projects/frontend/data-pipelines/gui/projects/ui/tsconfig.spec.json)
```

Comment:
Do we want to exclude all jsons under `data-pipelines`? If this is the case, it might be worth doing something like below (not sure about the exact regex pattern)

```
^projects/frontend/data-pipelines/.*json$
```

Reply:
not sure wewant to generalise that much, would prefer to pin the concrete files for exclusion at this point;
That would prevent adding a file that is excluded from the pre-commit hook - without the author realising it.

What are your thoughts on this?

Reply:
I'm fine with leaving it like this. Was just wondering if we can't use a shorter regex, but it seems like you want to make it more granular.